### DISCUSSION TITLE
Can't find changes done recently

### DISCUSSION THREAD

**Discussion by mjl38** [OpenSourcePFCLibraries/2019#22] [Q&A] [answered]
Hi Bruce,
Not too long ago, I did a change on behalve of John Fauss in the of_centimeterstopbunitsy() function of pfcapsrv.pbl, pfc_n_cst_pbunitconversion object.
I just downloaded the ZIP file, but cannot find this change.
https://github.com/OpenSourcePFCLibraries/2019/pull/15/commits/2591dc45f2aad946c47f3a2cce4d34f4f9f1411d

Neither these changes seem to be present:
https://github.com/OpenSourcePFCLibraries/2019/pull/14/commits/33a4253f932523ac27670592d9e94371d3ff30b9
https://github.com/OpenSourcePFCLibraries/2019/pull/14/commits/a4c0cf31e9305b6e4650ec4fe53f5143ff73b19d

If I do a search on "is_ClassName[1] =" I don't get any result anywhere. If I'm not mistaken, these values should be set in 
pfc_n_cst_platform:
constructor event:
------------------
ls_PBVMName = this.of_getPBVMName()
choose case ls_PBVMName
	case "pbvm.dll"  // pb2019 R3 (and most probably later versions too)
		is_ClassName[1] = "FNWND3" // mdi window
		is_ClassName[2] = "FNWNS3" // response window
	case "pbvm170.dll" // pb2017
		is_ClassName[1] = "FNWND3170" // mdi window
		is_ClassName[2] = "FNWNS3170" // response window
		is_ClassName[3] = "AfxMDIFrame100su" // pb2017 for "dockable MDI" 
	case "pbvm190.dll"  // pb2019 R1 / R2
		is_ClassName[1] = "FNWND3190" // mdi window
		is_ClassName[2] = "FNWNS3190" // response window
		// dockable MDI is now obsolete in pb2019 R2, commented:
		//	is_ClassName[3] = "AfxMDIFrame100su" // pb2019 for "dockable MDI", same as PB2017
	case "pbvm126" // pb 12.6
		is_ClassName[1] = "FNWND3126" // mdi window
		is_ClassName[2] = "FNWNS3126" // response window
	case "pbvm125" // pb 12.5
		is_ClassName[1] = "FNWND3125" // mdi window
		is_ClassName[2] = "FNWNS3125" // response window	
end choose

regards

**Comment by TABUllrich** [comment]
Hi Miguel,
have you checked it only in PBL or also in ws_objects? Maybe you have to do a Refresh to synchronize the PBL.
I remember that I've got your changes.
Regards

**Reply by mjl38** [reply]
Ha, that might be the explanation. I haven't checked ws_objects. But ...
Shouldn't the ZIP that people download already have all the latest changes?
Thanks, I'll check it out.

**Reply by mjl38** [reply]
I mean, most people will only download the ZIP and use it. I don't imagine them connecting to github doing a refresh to get the latest version.

**Reply by mjl38** [reply]
Okay, Ren'e Ullrich just replied me to ask in ws_objects. Indeed, the changes are there.

**Comment by bruce-armstrong** [comment, marked as answer]
I just created a new release that contains a new ZIP file with PBLs that should contain all the latest changes.

**Reply by mjl38** [reply]
That´s great news. Thank you Bruce!
